main_pullrequest.go

```diff
@@ -42,9 +42,14 @@ func processGitHubPullRequest(
 	log.Debugf("Processing pull request action %s", action)
 	switch action {
 	case "opened", "edited", "reopened", "synchronize", "ready_for_review":
+		msgDetails := "see <a href=\"https://console.cloud.google.com/kubernetes/" +
+			"deployment/us-east1/company-websites/default/test-runner-mender-io/logs?" +
+			"project=gp-kubernetes-269000\">logs<\a> for details."
```

Comment:
This does not render as you expect
![image](https://user-images.githubusercontent.com/3168644/191500060-44392234-1b30-4b33-aa94-ae672170fdb2.png)

Use markdown instead

Reply:
This should work with html. Here it's the typo - `\a` instead of `/a`.